webapp-django/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -244,8 +244,8 @@ def bugzilla_submit_url(report, bug_product):
         'product': bug_product,
         'op_sys': op_sys,
         'rep_platform': report['cpu_name'],
-        'cf_crash_signature': '[@ {}]'.format(report['signature']),
-        'short_desc': 'Crash in {}'.format(report['signature']),
+        'cf_crash_signature': u'[@ {}]'.format(report['signature']),
+        'short_desc': u'Crash in {}'.format(report['signature']),
```

Comment:
See updated commit. I tested it and clicked on the link and how the Summary is filled in correctly:
<img width="1415" alt="screen shot 2017-07-24 at 3 19 50 pm" src="https://user-images.githubusercontent.com/26739/28540645-153b0144-7084-11e7-9db1-137b75e40993.png">